pkgs/development/python-modules/mps-youtube/default.nix

```
@@ -4,14 +4,14 @@
 
 buildPythonPackage rec {
   pname = "mps-youtube";
-  version = "0.2.8";
+  version = "2020-01-28";
```

convention when not following an official release:
```suggestion
  version = "unstable-2020-01-28";
```